st2api/tests/unit/controllers/v1/test_kvps.py

```diff
@@ -107,65 +113,86 @@ def test_put(self):
     def test_put_with_scope(self):
         self.app.put_json('/v1/keys/%s' % 'keystone_endpoint', KVP,
                           expect_errors=False)
-        self.app.put_json('/v1/keys/%s?scope=system' % 'keystone_version', KVP_2,
+        self.app.put_json('/v1/keys/%s?scope=st2kv.system' % 'keystone_version', KVP_2,
```

Comment:
Ah, so the query param value changed as well.

Do we still handle "system" and "user" for backward compatibility reasons? If we do, we should add some explicit backward compatibility tests if we don't have any already...